distros/fedora/openalpr.spec

```diff
@@ -0,0 +1,125 @@
+Name:     openalpr
+Version:  2.3.0
+Release:  1%{?dist}
+Summary:  OpenALPR
+License:  AGPLv3
+URL:      https://github.com/openalpr/openalpr  
+source:   openalpr.tar.gz
```

Comment:
This should be `Source0:` instead of `source` and point to a location where the tarball can be downloaded from.